phpunit.xml.dist

```diff
@@ -25,7 +25,7 @@
         <env name="ZOOKEEPER_HOST" value="localhost" />
         <env name="COUCHBASE_HOST" value="localhost" />
         <env name="COUCHBASE_USER" value="Administrator" />
-        <env name="COUCHBASE_PASS" value="111111%40" />
+        <env name="COUCHBASE_PASS" value="111111@" />
```

Comment:
@alexandre-daubois @nicolas-grekas I am not really sure why this is required. Does the `CouchbaseCollectionAdapter` work differently than the `CouchbaseBucketAdapter`?

Reply:
That was the start point of #57192 😄 

This was changed to ensure that the Couchbase adapter is decoding special chars from password and you're able to put an encoded `@` in it. Special chars decoding is currently done on a case-by-case basis in each DSN adapter parser.

Why two classes? If I remember correctly, one of the adapter is deprecated and used for an old version of Couchbase, whereas the other one is the one to use in new application.

Reply:
Well, the decoding is not necessary in our integration tests as username and password are passed separately and are not part of the DSN. But that was also the case when we ran tests with the `CouchbaseBucketAdapter`. That's why I wondered if there was another difference which explains why we need this change to make tests pass.

Reply:
Not that I can remember 🤔

Reply:
I had some hard times debugging this adapter, it is complicated to test on a local machine. The fact it is unstable on the CI makes me wonder even more if this shouldn't be deprecated. The version supported by our adapter is old and no contributions seems planned to bring support for newer versions. I think the question of the deprecation arises.